React/Extensions/ReactExtensions.cs

```diff
@@ -10,27 +10,66 @@
 using Jering.Javascript.NodeJS;
 using React.Services;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
 
 namespace React.Extensions;
 
+public class ReactOptions
+{
+    public bool EnableExecutionTimeReactServiceRenderToStringAsync { get; set; } = false;
```

Comment:
There is nothing about logging execution time in property name, and this is what a decorator is doing.